mirgecom/artificial_viscosity.py

```diff
@@ -1,27 +1,31 @@
-r""":mod:`mirgecom.artificial_viscosity` Artificial viscocity for Euler.
+r""":mod:`mirgecom.artificial_viscosity` Artificial viscosity for hyperbolic
+conservation laws.
```

Comment:
I think docstyle will complain about this? I think this has to be a one-liner?

Reply:
Done in https://github.com/w-hagen/mirgecom/pull/18/commits/9b1dcc89545d8b489fb7aa841595578d1a57fc84